### PR TITLE
Fixes #8652: Add bundles to manage dry run of generic methods

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/8652

### CODE REVIEW

File: tree/20_cfe_basics/dry_run_lib.cf [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+# Body that can enable dry-run with a promise type granularity.
+# We dont put some promise type here because they should not be dry runned.
+# They are:
+#  - meta
+#  - vars
+#  - defaults
+#  - classes
+#  - methods
+#  - reports
+#  - ... and all promise types that are not fo the agent
```
Comment (amousset) [inline tree/20_cfe_basics/dry_run_lib.cf]: fo -> for

**Comment by amousset** [comment]
A test for this feature would be great, could be in another ticket.

**Comment by amousset** [comment]
The CFEngine test for default bodies could be a starting point: https://github.com/cfengine/core/blob/master/tests/acceptance/00_basics/03_bodies/default_files_action.cf.

File: tree/20_cfe_basics/dry_run_lib.cf [hunk 2]
```diff
@@ -0,0 +1,132 @@
+#####################################################################################
+# Copyright 2015 Normation SAS
```
Comment (ncharles) [inline tree/20_cfe_basics/dry_run_lib.cf]: copyright 2016, isn't it?

**Comment by peckpeck** [comment]
Commit modified

**Pull request merged**